src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -1309,6 +1329,70 @@
 short-summary: Retrieves latest restorable timestamp for the given sql container in given region.
 """
 
+# gremlin restorable resources
+helps['cosmosdb gremlin restorable-database'] = """
+type: group
+short-summary: Manage different versions of gremlin databases that are restorable in a Azure Cosmos DB account.
+"""
+
+helps['cosmosdb gremlin restorable-database list'] = """
+type: command
+short-summary: List all the versions of all the gremlin databases that were created / modified / deleted in the given restorable account.
+"""
+
+helps['cosmosdb gremlin restorable-graph'] = """
+type: group
+short-summary: Manage different versions of gremlin graphs that are restorable in a database of a Azure Cosmos DB account.
```

Comment:
`cosmosdb gremlin list-restorable-graph`

Reply:
Hi Yishi, we are doing this way so that we are compatible with existing Api types like SQL and MongoDB.